CMakeLists.txt

```
@@ -167,3 +167,25 @@ function(mbed_generate_bin_hex target)
         VERBATIM
     )
 endfunction()
+
+#
+# Parse toolchain generated map file for this `target` and display readable table format
+#
+function(mbed_parse_map_file target)
```

The function does more than just parsing.
```suggestion
function(mbed_generate_map_file target)
```